src/engine/plugins/tile.cpp

```diff
@@ -518,6 +532,7 @@ void encodeVectorTile(const datafacade::ContiguousInternalMemoryDataFacadeBase &
                                                    &max_datasource_id,
                                                    &used_line_ints](const FixedLine &tile_line,
                                                                     const std::uint32_t speed_kmh,
+                                                                    const std::uint32_t rate,
```

Comment:
Yeah, that's a vector tile encoding gotcha - I agree, it's a bit unintuitive if you haven't read the spec.  I'll rename these, now is as good a time as any.